src/components/Carousel/Carousel.vue

```diff
@@ -45,6 +45,27 @@
         →
       </button>
     </slot>
+
+    <!-- @slot Slot for Dots -->
+    <slot
+      v-if="!hideDots"
+      name="dots"
+      :go-to-slide="goToSlide"
+      :current-page="currentPage"
+      :dots-count="dotsCount"
+    >
+      <ol
+        class="vs-carousel__dots"
+        :aria-label="i18n.paginationNavigation">
+      >
+        <li
+          v-for="index in dotsCount"
+          :key="index"
+          :aria-current="index -1 === currentPage"
+          @click="goToSlide(index - 1)"
+        >⬤</li>
```

Comment:
We should keep HTML entity for this kind of bullets, otherwise  it might bring some cross-browser errors.